javaparser-core-testing/src/test/java/com/github/javaparser/javadoc/JavadocExtractorTest.java

```diff
@@ -42,7 +42,7 @@ void canParseAllJavadocsInJavaParser() throws FileNotFoundException {
 
     private void processFile(File file) throws FileNotFoundException {
         try {
-            CompilationUnit cu = parse(file, StandardCharsets.UTF_8);
+            CompilationUnit cu = parse(file);
```

Comment:
Why this?

Reply:
A recent PR moved the character encoding to ParserConfiguration, making parse methods with an explicit encoding redundant.